File: TrinhsGroup/TrinhsGroup/View/Model/AppSetting.swift
//
//  AppSetting.swift
//  TrinhsGroup
//
//  Created by long on 04/07/2022.
//

import Foundation

struct AppSetting: Codable {
    var status: String
    var code: String
    var currency_symbol: String
    var currency_position: String
    var thousand_separator: String
    var decimal_separator: String
    var number_of_decimals: String
}

File: kmp/shared/src/commonMain/kotlin/com/trinhsgroup/shared/model/AppSetting.kt
package com.trinhsgroup.shared.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class AppSetting(
    val status: String,
    val code: String,
    @SerialName("currency_symbol") val currencySymbol: String,
    @SerialName("currency_position") val currencyPosition: String,
    @SerialName("thousand_separator") val thousandSeparator: String,
    @SerialName("decimal_separator") val decimalSeparator: String,
    @SerialName("number_of_decimals") val numberOfDecimals: String
)

@Serializable
data class PasswordResetResponse(
    val message: String
)
